packages/library-export-default-webpack-plugin/package.json

```
@@ -6,8 +6,10 @@
 	"license": "GPL-2.0-or-later",
 	"keywords": [
 		"wordpress",
+		"gutenberg",
 		"webpack",
-		"webpack-plugin"
+		"plugin",
```

```suggestion
		"plugin",
		"webpack-plugin",
```

it looks like `webpack-plugin` is something people use, doesn't hurt having it